tests/test_resource.py

```diff
@@ -0,0 +1,106 @@
+#  Copyright 2019 XLAB d.o.o.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from itertools import cycle
+from unittest import mock
+
+import pytest
```

Comment:
@miha-plesko May I ask why this should be moved to the top? PEP8 clearly states that the order of imports is 1) standard library, 2)third party imports and 3) local imports.

Reply:
Ha, thanks @tadeboro I was somehow assuming pytest as builtin which is funny because we were discussing this today :=. @aljazkosir sorry, can you please move it back here in the middle, but there must be empty line after it. Thanks

Reply:
Moved to the middle :)